development-defaults.ini

```diff
@@ -0,0 +1,48 @@
+[enums]
+[[campsite]]
+asp1 = "Aspin Point 001"
+blake1 = "Bl Lakeside 001"
+blue5 = "Blue 5"
+brown4 = "Brown 4"
+brown5 = "Brown 5"
+brown7 = "Brown 7"
+brown16 = "Brown 16"
+brown17 = "Brown 17"
+brown20 = "Brown 20"
+purple11 = "Purple 11"
+purple12 = "Purple 12"
+purple14 = "Purple 14"
+purple15 = "Purple 15"
+purple16 = "Purple 16"
+purple17 = "Purple 17"
+purple19 = "Purple 19"
+purple20 = "Purple 20"
+purple27 = "Purple 27"
+purple28 = "Purple 28"
+purple33 = "Purple 33"
+purple34 = "Purple 34"
+purple35 = "Purple 35"
+purple37 = "Purple 37"
+purple38 = "Purple 38"
+red2 = "Red 2"
+red17 = "Red 17"
+tent49 = "Tent 49"
+tent50 = "Tent 50"
+tent51 = "Tent 51"
+tent52 = "Tent 52"
+tent53 = "Tent 53"
+tent54 = "Tent 54"
+tent55 = "Tent 55"
+tent56 = "Tent 56"
+tent57 = "Tent 57"
+tent58 = "Tent 58"
+tent59 = "Tent 59"
+tent75 = "Tent 75"
+yellow1 = "Yellow 1"
+yellow2 = "Yellow 2"
+yellow3 = "Yellow 3"
+yellow4 = "Yellow 4"
+yellow10 = "Yellow 10"
+yellow13 = "Yellow 13"
+yellow21 = "Yellow 21"
+bluejay = "Blue Jay"
```

Comment:
we'll need to add a bunch to this list, but can totally do it after this PR is merged no prob.  @bds002 take note, you and I will likely need to come up with the final list here.

I'm sure also that once we get onsite we'll have some updates to our list as well as we find out campsites are occupied, muddy, reserved, etc.  We'll probably need to keep open a tab with the report showing campsite assignments so far, and the google doc with the notes/quiet levels per site.